### DISCUSSION TITLE
New integrations, bug fix and Radarr image update

### DISCUSSION THREAD

**Discussion by trossr32** [trossr32/sonarr-radarr-lidarr-autosearch-browser-extension#69] [Announcements]
Add simkl and metacritic integrations.
Fix trakt integration.
Change Radarr image from the version 0 logo to the version 3 logo.

<hr /><em>This discussion was created from the release <a href='https://github.com/trossr32/sonarr-radarr-lidarr-autosearch-browser-extension/releases/tag/v2.2.0'>New integrations, bug fix and Radarr image update</a>.</em>

**Comment by PsychoManiac189** [comment]
Can you ad plex intergration? like this? 
![image](https://github.com/trossr32/sonarr-radarr-lidarr-autosearch-browser-extension/assets/95590670/1b1cf254-8480-4c64-94a0-7389972e8010)

**Comment by trossr32** [comment]
Closing as Plex integration will not be developed (at least not by me, happy for someone else to develop and maintain the feature)
